docs/adr/0008-secrets-management.md

```diff
@@ -0,0 +1,53 @@
+# GitOPS and Secrets Management
+
+* Status: accepted
+* Deciders: HumairAK, hemajv, 4n4nd, anishasthana, tumido, mhild
+
+Technical Story: [issue-1](https://github.com/open-infrastructure-labs/ops-issues/issues/3), [issue-2](https://github.com/operate-first/apps/issues/77)
+
+## Context and Problem Statement
+
+Following a GitOPs operating model requires us to describe our running system in Git. Since we operate on Openshift/K8s, we can do this easily by specifying our entire system via `yaml` manifests and storing them in GitHub. Things get a bit more complicated when trying to reconcile where to store confidential information like `secrets`. We use the word `secrets` here a bit loosely to capture all confidential manifests (e.g. k8s secrets, routes with certs, etc.).
```

Comment:
```suggestion
Following a GitOps operating model requires us to describe our running system in Git. Since we operate on Openshift/K8s, we can do this easily by specifying our entire system via `yaml` manifests and storing them in GitHub. Things get a bit more complicated when trying to reconcile where to store confidential information like `secrets`. We use the word `secrets` here a bit loosely to capture all confidential manifests (e.g. k8s secrets, routes with certs, etc.).
```